### ISSUE TITLE
YAML metadata?

### ISSUE THREAD

**Issue by danielbayley** [lucide-icons/lucide#1360]
What do we think of possibly having the metadata in YAML format, as opposed to JSON? It would be much nicer/easier/faster to deal with…

**Comment by karsa-mistmere** [comment]
Yeah, it's something I've also thought about, it would also significantly decrease merge conflicts, but I'm not totally sold on the idea.

**Comment by danielbayley** [comment]
> it would also significantly decrease merge conflicts

Yeah that too!
